src/js/index.js

```diff
@@ -0,0 +1,104 @@
+const $total = document.querySelector('#total');
+const $digits = document.querySelector('.digits');
+const $operations = document.querySelector('.operations');
+const $allClear = document.querySelector('.modifiers');
+
+let display = '';
+let buttons = [];
+let total = 0;
+
+const changeDisplay = (value) => {
+    display += value;
+    $total.innerHTML = display;
+};
+
+const calculate = () => {
+    for (let i = 0 ; i < 3 ; i++) {
+        if (i % 2 === 0) {
+            if (i === 0) {
+                total = parseInt(buttons[i], 10);
```

Comment:
ParseInt 대신에 
```suggestion
                total = +(buttons[i]);
```

로 써도 숫자가 됩니다!

Reply:
오, 처음 알았어요! 좋은 정보 감사합니다~